undertow/WFLY-17273_Global_tracking_mode_setting_in_Undertow.adoc

```diff
@@ -0,0 +1,100 @@
+= WFLY-17273 Global tracking-mode setting in Undertow
+:author:            Lin Gao
+:email:             lgao@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+There is an element: `tracking-mode` in `session-config` section of web descriptor, which is used to specify the session tracking mode, the available value can be: `COOKIE`, `URL`, `SSL`. Example in the web.xml is like:
+
+[source,xml]
+----
+<session-config>
+    <tracking-mode>COOKIE</tracking-mode>
+    <tracking-mode>URL</tracking-mode>
+</session-config>
+----
+
+When it is not defined, undertow uses `COOKIE` and falls over to `URL` if the browser does not allow cookie for the deployed web applications.
+
+When users want to have different set up for all web applications, they need to update the web.xml in each application. It would be great if undertow subsystem supports specifying the session tracking modes globally at the servlet container level.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17273[WFLY-17273]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1980[EAP7-1980]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
```

Comment:
please, don't forget to add your info here @gaol

Reply:
It refers to the vars defined at the top in the doc: `:author:` and `:email:`